docs/index.md

```diff
@@ -1439,6 +1439,9 @@ When the response has been written to the client, the context associated with th
 
 In the latter case, calling publish will return an error because we returned the HTTP response quickly.
 
+Note: From Go 1.21, there is a way to create a new context without cancel. WithoutCancel returns a copy of parent that is not canceled when parent is canceled.
```

Comment:
Can you please use this note syntax? https://github.com/teivah/100-go-mistakes/blob/master/docs/index.md?plain=1#L282-L284